ssh/config.json

```diff
@@ -14,7 +15,7 @@
   "hassio_role": "manager",
   "auto_uart": true,
   "ports": {
-    "22/tcp": 22
+    "22/tcp": null
```

Comment:
IMHO this change should have been a separate change that was advertised as a breaking change.   It should also have been described as "Disable network ssh access by default".

Reply:
Please use issues for discussions and bug reports instead, thanks 👍